Automattic/wp-calypso
# Plugins: Update cards design and layout

#### Changes proposed in this Pull Request

* Part of our aim to redesign the plugin browser in smaller pieces, see #43230
* Begin updating the card design and layout to include more information (added a short description, last updated, rating numbers, and downloads)
* Show the plugin cards aligned to a two-column grid instead of three.
* Add a flag for WP.com-supported plugins.

This design isn't finalized, bear with me. :)

#### Testing instructions

* Switch to this PR, sandbox the API, and apply D54040-code to your sandbox
* Navigate to `/plugins` on a simple site. You should see the updated card design.
* Switch to a Jetpack site. You should still see the updated card design.
* Make sure I haven't introduced visual regressions or weirdness.

See #43230
